ios/Classes/SwiftFlutterWebAuthPlugin.swift

```diff
@@ -71,6 +89,10 @@ public class SwiftFlutterWebAuthPlugin: NSObject, FlutterPlugin {
             result(FlutterMethodNotImplemented)
         }
     }
+
+    private func returnError(result: @escaping FlutterResult) {
```

Comment:
The function name is very generic "return error", but the actual error is very specific. I think that the best approach would be to make a `fileprivate extension` to `FlutterError` that adds a `aquireRootViewControllerFailed` getter that returns the error:

```swift
fileprivate extension FlutterError {
    var aquireRootViewControllerFailed: FlutterError {
        return FlutterError(code: "AQUIRE_ROOT_VIEW_CONTROLLER_FAILED", message: "Failed to aquire root view controller" , details: nil)
    }
```

Then you can remove this function and just do:

```diff
- returnError(result: result)
+ result(FlutterError.aquireRootViewControllerFailed)
```